docs/backend-api/how-to/how to get track points.md

```diff
@@ -0,0 +1,75 @@
+How to get track points for trips

```

Comment:
1. Missed `title` and `description` attributes at the begging.
1. Main title must started with `# `. 

Use [How to get session hash](https://raw.githubusercontent.com/SquareGPS/navixy-api/master/docs/backend-api/how-to/get-session-hash.md) as example.